CHANGELOG.md

```diff
@@ -5,22 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Gaprindashvili RC
+## Gaprindashvili-1
 
 ### Added
 - Tag mapping in graph refresh [(#64)](https://github.com/ManageIQ/manageiq-providers-openshift/pull/64)
-
-### Fixed
-- Added supported_catalog_types [(#177)](https://github.com/ManageIQ/manageiq-providers-openstack/pull/177)
-
-## Gaprindashvili Beta2
-
-### Fixed
-- Fix Inventory Collector has_required_role? [(#65)](https://github.com/ManageIQ/manageiq-providers-openshift/pull/65)
-
-## Gaprindashvili Beta1
```

Comment:
If these were public betas, IMHO it adds values to keep the granularity what changed in each of them?

BTW, can you specify date or version for "Initial changelog added" at bottom?  Otherwise, the whole section for changes in "Gaprindashvili-1" is not very useful because unclear since when...

Reply:
This is process we follow for all repos and releases.
Once we do a release, we roll up all betas and rcs into a final release.
I'll add the "Initial Changelog added" on the next changelog submission. I do have that in other repos and it seems overlooked this one.